03-Security-In-CICD/Readme.md

```diff
@@ -231,6 +238,86 @@ what we have seen in the case of ad hoc SVT runs:
 ![03_AzSDK_Logs](../Images/03_AzSDK_Logs.png)
 
 [Back to top...](Readme.md#contents)
+
+# Security Verification Tests (SVTs) in Jenkins pipeline
+	
+> Note : AzSDK Plugin requires PowerShell to be present on Jenkins Server. Therefore, the Plugin is currently supported for Windows machines only.	
+
+### Enable AzSDK extension for your Jenkins
+	
+Currently AzSDK CICD extension/Plugin has not been published on Jenkins repository, However you can use Jenkins Web UI to upload this Plugin([AzSDK_CICD_Jenkins_Plugin.hpi](Assets/AzSDK_CICD_Jenkins_Plugin.hpi) file) to Jenkins or place it in '$JENKINS_HOME/Plugins' location.
+	
+**Step to upload Plugin using Jenkins Web UI**
+	
+ Go to Home Page --> Manage Jenkins --> Manage Plugins -->  Select Advanced --> Upload Plugin file "[AzSDK_CICD_Jenkins_Plugin.hpi](Assets/AzSDK_CICD_Jenkins_Plugin.hpi)"
+
+![03_Upload_Plugin](../Images/03_Upload_Plugin.png)  
+ 
+![03_Install_Plugin](../Images/03_Install_Plugin.png)
+ <br/>Plugin is successfully imported. Now let's use Plugin to scan Azure Resources.  
+
+[Back to top...](Readme.md#contents)
+
+### Walkthrough
+This part assumes that you are familiar with Jenkins pipeline at a basic level. To get the basics and setup Jenkins you can refer [article](https://jenkins.io/doc/).
```

Comment:
To get the basics and setup Jenkins you can refer [article](https://jenkins.io/doc/).
  
May be "To explore more on Jenkins, refer: ----------------"

Reply:
Resolved.